crates/biome_formatter/src/buffer.rs

```diff
@@ -597,6 +608,41 @@ fn clean_interned(
     }
 }
 
+fn clean_best_fitting(
```

Comment:
We should document a bit of the logic. For example, we want to document what defines "being inside an expanded conditional content". (e.g. `is_in_expanded_conditional_content`)

Reply:
This function is for quick verification purposes, forgot to remove it. I've adjusted the logic related to `RemoveSoftLinesBuffer` a bit in https://github.com/biomejs/biome/pull/2500/commits/99330db0824c6344a7381824122fdb3d383f7fbb. Please take a look